backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/barchart/internal/BarChartColumn.kt

```diff
@@ -63,7 +63,7 @@ internal fun BarChartColumn(
   Column(
     horizontalAlignment = Alignment.CenterHorizontally,
     modifier = modifier
-      .semantics { contentDescription = model.accessibilityLabel }
+      .semantics { contentDescription = model.accessibilityLabel.toString() }
```

Comment:
as you're just changing this back to `toString` this does nothing. how the `TtsSpan` works (at least in the view world) is that you include the accessibility content as part of the text on a view and it'll read out the extra accessibility content as part of reading the content text. so if your text is August 11 the `TtsSpan` will include a span for this part saying that this is a date.

Reply:
Currently, there's no Tts support (TtsAnnotation) in Compose at all. I've changed the API to AnnotatedString to support it in the future, but for now yes, just `.toString()` method. Also, there's no way to use TtsSpan from View and no direct access to View accessibility node from Compose.

Reply:
yeah what i'm saying is: using an annotated string here won't work for the content description (as that's just a string) - the `TtsSpan` needs to be part of the content text directly, not a separate content description, so changing the content description to an annotated string won't help - we'd have to change the content text to be an `AnnotatedString` and not have an `accessibilityLabel` at all.

Reply:
We don't have content text. We only got the title, subtitle, and value text. If we make any of these parameters AnnotatedString it will make the API more complicated. Also, it's not clear why it's the title (and not subtitle, or value) that should play the main role as an accessibility label provider.

When TttsAnnotation will be available we can inside set it as a text to any component we want. For instance, we can add TtsAnnotation to the title, but keep the title text the same as it was. But those are the implementation detail